nautobot_golden_config/tests/test_basic.py

```diff
@@ -1,15 +1,26 @@
 """Basic tests that do not require Django."""
-import os
 import unittest
-
+import os
 import toml
 
+from nautobot_golden_config import __version__ as project_version
+
+
+class TestVersion(unittest.TestCase):
+    """Test Version is the same."""
+
+    def test_version(self):
+        """Verify that pyproject.toml version is same as version specified in the package."""
+        parent_path = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+        poetry_version = toml.load(os.path.join(parent_path, "pyproject.toml"))["tool"]["poetry"]["version"]
+        self.assertEqual(project_version, poetry_version)
+
 
 class TestDocsPackaging(unittest.TestCase):
     """Test Version in doc requirements is the same pyproject."""
 
     def test_version(self):
-        """Verify that pyproject.toml dev dependencies have the same versions as in the docs requirements.txt."""
+        """Verify that pyproject.toml dev dependecies have the same versions as in the docs requirements.txt."""
```

Comment:
@snaselj can you fix this in the template?

Reply:
Fixed here, will fix it in the template.